portfolio/src/main/webapp/index.html

```diff
@@ -14,7 +14,7 @@
     <div class="navbar"> 
         
         
-            <div style="float:left;">
+            <div  class="logoF"style="float:left;">
```

Comment:
What does this do?

Reply:
i just gave it a class when i was testing